starknet_py/net/full_node_client.py

```diff
@@ -93,6 +95,44 @@ async def get_block_traces(
     ) -> BlockTransactionTraces:
         raise NotImplementedError()
 
+    async def get_events(
+        self,
+        from_block_number: Optional[Union[int, Tag]] = None,
+        from_block_hash: Optional[Union[Hash, Tag]] = None,
+        to_block_number: Optional[Union[int, Tag]] = None,
+        to_block_hash: Optional[Union[Hash, Tag]] = None,
+        contract_address: Optional[Hash] = None,
+        keys: Optional[List[Hash]] = None,
+    ) -> Events:
+        # pylint: disable=too-many-arguments
+        params = {
+            "chunk_size": 1024,
+            "from_block": get_block_identifier(from_block_hash, from_block_number),
+            "to_block": get_block_identifier(to_block_hash, to_block_number),
+        }
+
+        if contract_address:
```

Comment:
```suggestion
        if contract_address is not None:
```
I prefer we used explicit None checks here and below